docs/proposals/pepxxx_wheel_variant_support.md

```diff
@@ -183,6 +765,103 @@ Several alternative approaches were considered and ultimately rejected:
 
 ![pytorch selector](../assets/images/pytorch_variant_selector.webp)
 
+### Wheel Variants
+
+#### Wheel filename
+
+- Adding the variant label as a third component (before the build tag) with additional `~` characters. This approach
+  was ultimately rejected, as adding it as a last component made it easier to distinguish different variants,
+  and achieved the same goals.
+
+- Using labels in addition to the variant hash. This approach was rejected because it caused unnecessary increase
+  of filename length. As the specification evolved and made it unnecessary to include the hash in the filename,
+  it was suggested to replace it with a human-readable label instead.
+
+- Automatically generating human-readable labels by providers. This idea did not fit well with very limited variant
+  label length.
+
+- Using a distinct character (such as `+`) as a variant label separator. This would make wheel variants even more
+  distinct from regular wheels, and make it easier to adjust the current filename processing algorithms, by avoiding
+  the ambiguity of a 6-component filename (that could either mean build tag or variant label being present).
+  The proposal relies on the variant label being interpreted as part of the platform tag by pre-variant implementations,
+  and therefore rejected based on unsupported platform. However, this approach does not work correctly if the wheel
+  uses multiple platform tags, e.g. `platform1.platform2+label` could be selected if `platform1` matched.
+
+#### Variant properties
+
+- Originally, only a single value was permitted for a property. This assumed that variants designate a specific
+  property of the wheel (e.g. a CPU version it was built for), while the provider plugins indicate which of these
+  properties are supported by the system. However, during discussion the need for an opposite approach was indicated,
+  where the wheel designates its compatibility (e.g. as a multiple compatible runtime versions), and the plugin verifies
```

Comment:
Can you add the specific example here?

Reply:
I think @DEKHTIARJonathan would know better here. It was his idea, after all.